### PR TITLE
Bug 1799051: Add OLM annotations for ui hints

### PR DESCRIPTION
**Pull request by jcantrill**
https://bugzilla.redhat.com/show_bug.cgi?id=1799051

### CODE REVIEW

File: manifests/4.4/cluster-logging.v4.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -299,15 +301,15 @@ spec:
                   - name: OPERATOR_NAME
                     value: "cluster-logging-operator"
                   - name: ELASTICSEARCH_IMAGE
-                    value: "quay.io/openshift/origin-logging-elasticsearch5:latest"
+                    value: registry.svc.ci.openshift.org/ocp/4.4:logging-elasticsearch5
```
Comment (ewolinetz) [inline manifests/4.4/cluster-logging.v4.4.0.clusterserviceversion.yaml]: isn't the old pattern what will get replaced for us by the CI process?

Comment (jcantrill) [reply manifests/4.4/cluster-logging.v4.4.0.clusterserviceversion.yaml]: Reverting..  this is not supposed to be here

**Pull request merged**